ui/core/src/assets.ethereum.mainnet.json

```diff
@@ -259,7 +239,7 @@
       "transferLimit": "460405156537753272320"
     },
     {
-      "address": "0xb83cd8d39462b761bb0092437d38b37812dd80a2",
+      "address": "0xc944e90c64b2c07662a292be6244bdf05cda44a7",
```

Comment:
Why was this address changed?

Reply:
See Sifchain Launch Token Google Doc: https://docs.google.com/spreadsheets/d/12yVNc7VCNdR3hzWsr5NJhfxBGRAduUsLSeYpVmwN-qg/edit#gid=0&range=D19

Reply:
+1

Reply:
Where did we get that grt token address from? Was that entered into the spreadsheet like that?

Reply:
GRT https://etherscan.io/token/0xc944e90c64b2c07662a292be6244bdf05cda44a7

Reply:
That's how it was this morning when I checked over everything